frontend/src/modules/root/page/Home/index.tsx

```diff
@@ -1,5 +1,6 @@
 import Btn from 'common/components/Button';
 import Footer from 'modules/root/components/Footer';
+import Navbar from 'modules/root/components/Navbar/index';
```

Comment:
this import look strange 
it should look like this `modules/root/components/Navbar`